.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,7 +16,7 @@
 
 ### Fixed Issues (if relevant)
 <!---
-    If relevant, please provide a list of fixed issues in the format magento/magento-cloud-patches#<issue_number>.
```

Comment:
@shiftedreality  This is the Patches repo, so I think patches is correct here.  So I just added the PR template updates to ece-tools, patches, and components to facilitate dev work for release note and doc updates.

Reply:
yeap, haven't notice that